src/components/utils/test/thread_test.cc

```diff
@@ -54,125 +55,355 @@ sync_primitives::Lock test_mutex_;
 
 }  // namespace
 
-class TestDelegate : public threads::ThreadDelegate {
- public:
-  TestDelegate() : check_value_(false) {}
-  void threadMain() {
-    AutoLock test_lock(test_mutex_);
-    check_value_ = true;
-    cond_var_.NotifyOne();
-  }
-  void exitThreadMain() {}
-
-  bool check_value() const {
-    return check_value_;
-  }
-
- private:
-  bool check_value_;
+class TestDelegate: public threads::ThreadDelegate {
+public:
+	TestDelegate() :
```

Comment:
@pvvasilev
Formatting in whole file seems to be broken. Use "./tools/infrastructure/check_style.sh --fix" or install git hook.